packages/stream-model-handler/src/model-handler.ts

```diff
@@ -125,6 +128,9 @@ export class ModelHandler implements StreamHandler<Model> {
 
     if (state.content.schema !== undefined) {
```

Comment:
```suggestion
    if (state.content.schema) {
```

IMO this should work too, and is a bit more idiomatic.